flux-refineck/src/fixpoint.rs

```diff
@@ -174,11 +185,15 @@ where
         let constants = self
             .const_map
             .values()
-            .map(|const_info| (const_info.name, fixpoint::Sort::Int))
+            .map(|const_info| {
+                fixpoint::ConstInfo {
+                    name: const_info.name,
+                    orig: const_info.sym,
+                    sort: const_info.sort.clone(),
+                }
+            })
```

Comment:
Use `into_values` instead of `values` to avoid cloning.

```suggestion
            .into_values()
            .map(|const_info| {
                fixpoint::ConstInfo {
                    name: const_info.name,
                    orig: const_info.sym,
                    sort: const_info.sort,
                }
            })
```